ara/plugins/callback/ara_default.py

```diff
@@ -848,6 +873,9 @@ def _get_user(self):
         if not self.record_user:
             return user
 
+        if self.record_user_name:
+            return self.record_user_name
```

Comment:
I wonder for both the user and the controller, should we move this before the first if ?

This way setting ``ARA_RECORD_USER_NAME`` would be sufficient without requiring ``ARA_RECORD_USER`` to be true.

Reply:
On second thought, these settings already default to true so it would only matter if someone explicitly set them to false.

It's probably fine like this, what do you think ?

Reply:
Yes, that was my thought. It's enabled by default, and if someone wants to disable it, then disabling would apply to the custom specification as well.